cli/src/cmd/forge/script/broadcast.rs

```diff
@@ -0,0 +1,260 @@
+use crate::cmd::ScriptSequence;
+
+use ethers::{
+    prelude::{Provider, SignerMiddleware},
+    providers::Middleware,
+    signers::Signer,
+    types::{
+        transaction::eip2718::TypedTransaction, Address, Chain, Eip1559TransactionRequest,
+        TransactionReceipt, TransactionRequest, U256,
+    },
+};
+
+use std::collections::BTreeMap;
+
+use super::*;
+
+impl ScriptArgs {
+    pub async fn send_transactions(
+        &self,
+        deployment_sequence: &mut ScriptSequence,
+    ) -> eyre::Result<()> {
+        // The user wants to actually send the transactions
+        let mut local_wallets = vec![];
+        if let Some(wallets) = self.wallets.private_keys()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.interactives()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.mnemonics()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.keystores()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
```

Comment:
we should discuss - but im leaning towards removing multiwallet support

Reply:
what do you mean? A single sender per function?? that seems brutal for UX.

Reply:
I'm curious on why? On [forgedeploy](https://github.com/joshieDo/foundrydeploy), i'm often jumping between trezor and local wallet